package.json

```diff
@@ -222,6 +222,21 @@
                 "command": "appService.InstallCosmosDBExtension",
                 "title": "Install Cosmos DB Extension",
                 "category": "Azure App Service"
+            },
+            {
+                "command": "appService.revealConnection",
+                "title": "Reveal in Cosmos DB extension",
+                "category": "Azure App Service"
+            },
+            {
+                "command": "appService.AddCosmosDBConnection",
+                "title": "Add New Connection...",
+                "category": "Azure App Service"
+            },
+            {
+                "command": "appService.DeleteCosmosDBConnection",
```

Comment:
nit: Rename the command to 'RemoveCosmosDBConnection' as well (and the underlying code)